src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -74,6 +74,5 @@ export default Object.freeze({
   evssUploadLimit150Mb: 'evss_upload_limit_150mb',
   searchRepresentative: 'search_representative',
   gibctStateSearch: 'gibct_state_search',
-  eduFormOmbAndExpiration: 'edu_form_omb_and_expiration',
```

Comment:
In this case, we want to keep the flag for future use. We just want to remove the current usage.

Reply:
10-4